Formula/node.rb

```diff
@@ -56,30 +56,11 @@ def install
 
     if build.with? "npm"
       resource("npm").stage buildpath/"npm_install"
-
-      # make sure npm can find node
-      ENV.prepend_path "PATH", bin
```

Comment:
> That means those formulae will need changing too :(

I _think_ we can _probably_ handle this on the `brew` side of things, but, I'm not sure it's going to be pretty 😕.

> What happens when someone tries to upgrade npm in place at that point?

We should be fine because we keep the existing `npm` in `libexec` forever and that's the one we point to in `language/node`. We pretty much choose to ignore what the user does with `npm` outside of the Cellar as a kind of sanity-protection.